src/Widget/MessageFlash.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Widget;
+
+use App\Widget\Message;
```

Comment:
Not used.

Reply:
https://github.com/terabytesoft/app/blob/b2d416d8c380cc21a7c5c5a3de04c09da59f2543/src/Widget/MessageFlash.php#L26

Reply:
Yes but it's in the same namespace thus the import isn't needed.